api/src/middlewares/services/EmailService/oathSubmissionTemplate.ts

```diff
@@ -0,0 +1,8 @@
+const template = `<ul>
```

Comment:
instead of adding a bundler with some plugins that can import html or handlebars, I've just made this a string instead.